### PR TITLE
Add eleventy topic

### PR DESCRIPTION
**Pull request by binyamin**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).
\* I triage issues for Eleventy. As such, I am a member of the GitHub organization. However, I do not maintain any part of the codebase.
### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

It seems to me that Eleventy has become very popular recently. It would be a shame if developers who are only familiar with JavaScript were forced to use the Ruby-based Jekyll simply because they didn't know any better.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/eleventy/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+aliases: 11ty
+created_by: Zach Leatherman
+display_name: Eleventy
+github_url: https://github.com/11ty
+logo: eleventy.png
+related: jekyll
```
Comment (rmathew1992) [inline topics/eleventy/index.md:7]: For related, could we change this to words that describe what the project does as opposed to similar products?

**Review by rmathew1992 — COMMENTED** [review]
One small comment, after which I think this looks good!

**Comment by binyamin** [comment]
> One small comment,, and the tests need to pass after which I think this looks good!

Thanks. After looking at the tests, I don't think they're related to this PR. https://github.com/github/explore/pull/1762/checks?check_run_id=1125719247

File: topics/eleventy/index.md (hunk 1 above)
Comment (binyamin) [reply topics/eleventy/index.md:7]: How about `static-site-generator`?

**Comment by rmathew1992** [comment]
Oh thanks for finding that! I opened a PR to fix it: https://github.com/github/explore/pull/1768 Once it gets merged, I'll merge yours in.

**Review by rmathew1992 — APPROVED** [review]

**Pull request merged**